package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "zen-audio-player-electron",
+  "version": "0.1.0",
+  "main": "main.js",
+  "dependencies": {
+    "electron-prebuilt": "^1.2.7"
+  },
+  "devDependencies": {
+    "eslint": "^3.1.1",
+    "eslint-config-google": "^0.6.0",
```

Comment:
All of them actually. We should only need `eslint` as a dev dependency at this point. They're not being used by the eslint config file, right?